src/main/js/newNpcsFromTemplateForm.tsx

```diff
@@ -39,8 +48,118 @@ const styles = ({ palette, spacing }: Theme) => createStyles({
     gridList: {
         width: "100%",
     },
+    input: {
+        display: 'flex',
+        padding: 0,
+    },
+    valueContainer: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        flex: 1,
+        alignItems: 'center',
+        overflow: 'hidden',
+    },
+    chip: {
+        margin: `${spacing.unit / 2}px ${spacing.unit / 4}px`,
+    },
+    chipFocused: {
+        backgroundColor: emphasize(palette.type === 'light' ? palette.grey[300] : palette.grey[700], 0.08,),
+    },
+    placeholder: {
+        position: 'absolute',
+        left: 2,
+        fontSize: 16,
+    },
 });
 
+const sizeSuggestions = [{value: 'Tiny', label: 'Tiny'}, {value: 'Small', label: 'Small' }, {value: 'Medium', label: 'Medium' }, {value: 'Large', label: 'Large' }, {value: 'Huge', label: 'Huge' }, {value: 'Gargantuan', label: 'Gargantuan' }];
+
+const movementSuggestions = [{value: 'Walk', label: 'Walk'}, {value: 'Burrow', label: 'Burrow' }, {value: 'Climb', label: 'Climb' }, {value: 'Fly', label: 'Fly' }, {value: 'Swim', label: 'Swim' }];
+
+const typeSuggestions = [{value: 'Aberration', label: 'Aberration'}, {value: 'Beast', label: 'Beast' }, {value: 'Celestial', label: 'Celestial' }, {value: 'Construct', label: 'Construct' },
+{value: 'Dragon', label: 'Dragon' }, {value: 'Elemental', label: 'Elemental' }, {value: 'Fey', label: 'Fey' }, {value: 'Fiend', label: 'Fiend' }, {value: 'Giant', label: 'Giant' },
+{value: 'Humanoid', label: 'Humanoid' }, {value: 'Monstrosity', label: 'Monstrosity' }, {value: 'Ooze', label: 'Ooze' }, {value: 'Plant', label: 'Plant' }, {value: 'Undead', label: 'Undead' }];
+
+const alignmentSuggestions = [{value: 'Lawful Good', label: 'Lawful Good'}, {value: 'Neutral Good', label: 'Neutral Good' }, {value: 'Chaotic Good ', label: 'Chaotic Good ' },
```

Comment:
There is a slightly larger list here:
  ANY_NON_GOOD,
  ANY_NON_LAWFUL,
  ANY_CHAOTIC,
  ANY_EVIL,
  ANY,
  UNALIGNED,
  CHAOTIC_EVIL,
  CHAOTIC_NEUTRAL,
  CHAOTIC_GOOD,
  NEUTRAL_EVIL,
  NEUTRAL,
  NEUTRAL_GOOD,
  LAWFUL_EVIL,
  LAWFUL_NEUTRAL,
  LAWFUL_GOOD

Reply:
@Mandalorian007 
Good catch done